indexer/feature_utils.cpp

```diff
@@ -503,4 +503,41 @@ string FormatElevation(string_view elevation)
   return {};
 }
 
+constexpr char const * kWlan = "wlan";
+constexpr char const * kWired = "wired";
+constexpr char const * kTerminal = "terminal";
+constexpr char const * kYes = "yes";
+constexpr char const * kNo = "no";
+
+string DebugPrint(Internet internet)
+{
+  switch (internet)
+  {
+  case Internet::No: return kNo;
+  case Internet::Yes: return kYes;
+  case Internet::Wlan: return kWlan;
+  case Internet::Wired: return kWired;
+  case Internet::Terminal: return kTerminal;
+  case Internet::Unknown: break;
+  }
+  return {};
+}
+
+Internet InternetFromString(std::string_view inet)
+{
+  if (inet.empty())
+    return Internet::Unknown;
+  if (inet.find(kWlan) != string::npos)
```

Comment:
If constants are string_view, can == be used instead of find? Which OSM value ases are handled by find?

Reply:
Isn't it a bug? Should several values be stored in the data?

Reply:
Normal for OSM